domain/state_test.go

```diff
@@ -49,16 +49,18 @@ func (s *stateSuite) TestStateBasePrepare(c *gc.C) {
 	stmt1, err := base.Prepare("SELECT name AS &M.* FROM sqlite_schema", sqlair.M{})
 	c.Assert(err, gc.IsNil)
 	// Validate prepared statement works as expected.
+	var name string
```

Comment:
We should probably have these as `any` to allow the type assertion to be performed by the checker.